vec/vec-v113/src/main/resources/generate-xjb-from-vec-model.xsl

```diff
@@ -42,9 +42,7 @@
         <xsl:variable name="parents" select="key('parents',@xmi:id)/.."/>
         <xsl:if test="$parents or $outgoing-refs">
             <jxb:bindings>
-                <xsl:attribute name="node">//xs:complexType[@name='<xsl:apply-templates select="."
-                        mode="create-name"/>']
-                </xsl:attribute>
+                <xsl:attribute name="node">//xs:complexType[@name='<xsl:apply-templates select="." mode="create-name"/>']</xsl:attribute>
```

Comment:
nit: Perhaps the <xsl:apply-templates [...]</xsl:attribute> should be on its own (like in other cases).